hypothesis-python/tests/pandas/test_data_frame.py

```diff
@@ -241,3 +241,11 @@ def test_uniqueness_does_not_affect_other_rows_2():
 def test_will_fill_missing_columns_in_tuple_row(df):
     for d in df['A']:
         assert d == 7
+
+
+@given(pdst.data_frames(
+    index=pdst.range_indexes(10, 10),
+    columns=[pdst.column(elements=st.integers(0, 9), fill=None, unique=True)],
+))
+def test_cen_generate_unique_columns(df):
```

Comment:
This test is a bit mysterious, and only checks for uniqueness pretty indirectly. It also seems needlessly expensive, and I'm not sure how it connects up with the rest of the PR. What's going on here?

Reply:
It's not connected except by coverage of [this `else` clause](https://github.com/HypothesisWorks/hypothesis/blob/208d4bdeed40051a4b6f3edfd8538db947de0acd/hypothesis-python/src/hypothesis/extra/pandas/impl.py#L572-L578) being kinda flaky.  Smaller columns didn't reliably cover it either, but very happy for advice on how to do it better!